pkg/webhook/admission/pod/batcher_injector.go

```diff
@@ -41,6 +41,9 @@ type BatcherConfig struct {
 	CpuLimit      string `json:"cpuLimit"`
 	MemoryRequest string `json:"memoryRequest"`
 	MemoryLimit   string `json:"memoryLimit"`
+	MaxBatchSize  string `json:"maxBatchSize"`
+	MaxLatency    string `json:"maxLatency"`
+	Timeout       string `json:"timeout"`
```

Comment:
We also need to update the ConfigMap and helm config

https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml#L499
https://github.com/kserve/kserve/blob/master/charts/kserve-resources/templates/configmap.yaml#L429

Reply:
Thanks. Updated